spec/libraries/aws_engine_spec.rb

```diff
@@ -75,6 +75,25 @@ def initialize
       end
     end
   end
+
+  module EC2
+    class DummyClient
+      class DummyCredential
+        def access_key_id
+          'iam_access_key'
+        end
+        def secret_access_key
+          'iam_secret_access_key'
+        end
+        def session_token
+          'iam_session_token'
+        end
+      end
+      def config
```

Comment:
__[Reek]__

[UtilityFunction] Aws::EC2::DummyClient#config doesn't depend on instance state (maybe move it to another class?)


<!-- Comment by [SideCI](https://sideci.com) -->